sycl/doc/EnvironmentVariables.md

```diff
@@ -28,6 +28,7 @@ subject to change. Do not rely on these variables in production code.
 | SYCL_DEVICELIB_NO_FALLBACK | Any(\*) | Disable loading and linking of device library images |
 | SYCL_PI_LEVEL0_MAX_COMMAND_LIST_CACHE | Positive integer | Maximum number of oneAPI Level Zero Command lists that can be allocated with no reuse before throwing an "out of resources" error. Default is 20000, threshold may be increased based on resource availabilty and workload demand. |
 | SYCL_PI_LEVEL0_DISABLE_USM_ALLOCATOR | Any(\*) | Disable USM allocator in Level Zero plugin (each memory request will go directly to Level Zero runtime) |
+| SYCL_PI_LEVEL_ZERO_BATCH_SIZE | Positive integer | Sets a preferred number of commands to batch into a command list before executing the command list. Values 0 and 1 turn off batching. Default is 4. |
```

Comment:
I'd add that this is really a max number of commands, and that execution can start earlier than that, e.g. at each blocking command.

Reply:
That is why I used said "Sets a preferred number".  I don't intend this to actually be a max (although it is for this particular change-set).  I think heuristics in the future might move it either up or down.  So I really want the semantic of this env var to be setting the user preference, not a specific limit.

Reply:
OK